json-format.md

```diff
@@ -51,6 +51,14 @@ the root of the extension MUST be serialized as a top-level JSON property. There
 were many reason for this design decision and they are covered in more detail in
 the [Primer](primer.md#json-extensions).
 
+For clarity, extension attributes are serialized using the same rules as
```

Comment:
I think this is a dup of the previous para - bad rebase maybe?

Reply:
yep - fixed rebase ;)